scripts/generate-releases.sh

```diff
@@ -66,6 +66,7 @@ validate_input() {
 generate_release_resources() {
     release_name="$(date +"%Y-%m-%d")-$(git rev-parse --short "$COMMIT")-${ENVIRONMENT}"
     release_name_short="$(date +"%Y%m%d")-$(git rev-parse --short "$COMMIT")-${ENV_SHORT}" # save space for retry suffix
+    whitelist_file="$ROOT_DIR/release-history/.whitelist.yaml"
     out_file="$ROOT_DIR/release-history/${release_name}.yaml"
     echo "Writing resources to ${out_file} ..." >&2
     while IFS= read -r line
```

Comment:
You added blank lines specifically where I pointed but the function does not look good yet. It's fair that everybody's aesthetic views are different, but doesn't it look weird?

<img width="1158" height="590" alt="image" src="https://github.com/user-attachments/assets/c052795c-b434-4068-9bc7-e623dae5977f" />

The function deserves more love, in my opinion. Let's add a blank line before the loop
```suggestion
    echo "Writing resources to ${out_file} ..." >&2

    while IFS= read -r line
```